src/AdsClient.php

```diff
@@ -250,40 +257,31 @@ public function getMessageIds(?string $blockId = null): GetMessageIdsResponse
     /**
      * Transfers funds to many accounts.
      *
-     * @param array $wires array of wires. Each entry is pair: account address => amount in clicks.
-     *                     Example: ['0001-00000000-XXXX'=>200,'0001-00000001-XXXX'=>10]
-     *
-     * @return SendManyResponse
-     *
-     * @throws CommandException
+     * @param SendManyCommand $command SendManyCommand
+     * @param bool $isDryRun if true, transaction won't be send to network
+     * @return SendTransferResponse
      */
-    public function sendMany(array $wires): SendManyResponse
+    public function sendMany(SendManyCommand $command, bool $isDryRun = false): SendTransferResponse
     {
-        $command = new SendManyCommand($wires);
         $this->prepareTransaction($command);
-        $response = $this->driver->executeCommand($command);
+        $response = $this->driver->executeTransaction($command, $isDryRun);
 
-        return new SendManyResponse($response->getRawData());
+        return new SendTransferResponse($response->getRawData());
     }
 
     /**
      * Transfers funds to one account.
      *
-     * @param string $address address to which funds will be transferred
-     * @param int $amount transfer amount in clicks
-     * @param null|string $message optional message, 32 bytes hexadecimal string without leading 0x (64 characters)
-     *
-     * @return SendOneResponse
-     *
-     * @throws CommandException
+     * @param SendOneCommand $command SendOneCommand
+     * @param bool $isDryRun if true, transaction won't be send to network
+     * @return SendTransferResponse
      */
-    public function sendOne(string $address, int $amount, ?string $message = null): SendOneResponse
+    public function sendOne(SendOneCommand $command, bool $isDryRun = false): SendTransferResponse
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "sendMany" on line 264. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS4144)